common/components/BalanceSidebar/AccountInfo.scss

```diff
@@ -2,6 +2,17 @@
 @import 'common/sass/mixins';
 
 .AccountInfo {
+  &-copy-icon{
+    width: 15px;
+    color: black;
+    &-copied{
+      color: red;
```

Comment:
Can you change `color: black;` and `color: red;` with some of our existing colors in `common/sass/variables/colors.scss`? The file's already imported in here. I'd suggest `$gray-light` by default, `$text-color` on hover, and `$brand-success` or `$brand-primary` for the copied state. We usually save red for errors.